be/src/exec/vectorized/file_scan_node.cpp

```diff
@@ -285,14 +289,22 @@ void FileScanNode::_scanner_worker(int start_idx, int length) {
             // remove range desc with empty file
             TBrokerScanRange new_scan_range(scan_range);
             new_scan_range.ranges.clear();
-            for (const TBrokerRangeDesc& range_desc : scan_range.ranges) {
+            for (TBrokerRangeDesc range_desc : scan_range.ranges) {
                 // file_size is optional, and is not set in stream load and routine load,
                 // so we should check file size is set firstly.
                 if (range_desc.__isset.file_size && range_desc.file_size == 0) {
                     continue;
                 }
+                if(skip_headline) {
+                    VLOG_QUERY << "skip is true";
+                    int num_fields_in_csv = range_desc.num_of_columns_from_file;
+                    range_desc.__set_start_offset(num_fields_in_csv);
```

Comment:
why set start offset to num_fields_in_csv?

Reply:
In order to skip the headline if the first line is the attributions name.